lib/SILOptimizer/Analysis/ClosureScope.cpp

```diff
@@ -0,0 +1,196 @@
+//===--- ClosureScope.cpp - Closure Scope Analysis ------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// Implementation of ClosureScopeAnalysis.
+//
+//===----------------------------------------------------------------------===//
+
+#define DEBUG_TYPE "closure-scope"
+
+#include "swift/SIL/SILModule.h"
+#include "swift/SILOptimizer/Analysis/ClosureScope.h"
+
+namespace swift {
```

Comment:
Question. Why use a swift namespace here instead of using an anonymous namespace + using namespace swift?